tools/CoherenceBuild/CoherenceBuild.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp1.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <RuntimeFrameworkVersion>1.0.5</RuntimeFrameworkVersion>
```

Comment:
I don't think anyone's tried to update it. 

We could update to 2.0.0, but we'll hit the exact same issue once we upgrade to 2.1.0-*. This is why it would be better to make it a RepoTask, or to specify required runtimes in the project.